blackjack/api/views.py

```diff
@@ -20,8 +20,17 @@ def __init__(self):
         random.shuffle(self.deck_cards)
 
 
+@api_view(["GET", "POST", "PUT"])
+def deal(request, id):
+    data = json.loads(request.body)
```

Comment:
good intuition (we will want to read some data from the `request`, eventually)

for now, ignore the `request` object and manually assign values to these variables from the Game instance